cmake/modules/AddSwift.cmake

```diff
@@ -1833,7 +1840,21 @@ function(_add_swift_executable_single name)
       BINARY_DIR ${SWIFT_RUNTIME_OUTPUT_INTDIR}
       LIBRARY_DIR ${SWIFT_LIBRARY_OUTPUT_INTDIR})
 
+  # On Windows, libraries are prefixed with "lib", so we need to adjust FAT_LIBRARIES for this.
```

Comment:
Interesting - I'm fairly sure it just links as a library. Without this change, I get an error that goes something like `cannot find library swiftReflection.lib`. However, `libSwiftReflection.lib` *does* exist, and everything links correctly.